pkg/sync/v14/data/ConfigMap/openshift-azure-logging/omsagent-rs-config.yaml

```diff
@@ -0,0 +1,207 @@
+apiVersion: v1
+data:
+  kube.conf: |-
+    # Fluentd config file for OMS Docker - cluster components (kubeAPI)
+     #fluent forward plugin
+     <source>
+      type forward
+      port "#{ENV['HEALTHMODEL_REPLICASET_SERVICE_SERVICE_PORT']}"
+      bind 0.0.0.0
+      chunk_size_limit 4m
+     </source>
+     #Kubernetes pod inventory
+     <source>
+      type kubepodinventory
+      tag oms.containerinsights.KubePodInventory
+      run_interval 60
+      log_level debug
+     </source>
+     #Kubernetes events
+     <source>
+      type kubeevents
+      tag oms.containerinsights.KubeEvents
+      run_interval 60
+      log_level debug
+      </source>
+     #Kubernetes Nodes
+     <source>
+      type kubenodeinventory
+      tag oms.containerinsights.KubeNodeInventory
+      run_interval 60
+      log_level debug
+     </source>
+     #Kubernetes health
+     <source>
+      type kubehealth
+      tag kubehealth.ReplicaSet
+      run_interval 60
+      log_level debug
+     </source>
+     #cadvisor perf- Windows nodes
+     <source>
+      type wincadvisorperf
+      tag oms.api.wincadvisorperf
+      run_interval 60
+      log_level debug
+     </source>
+     <filter mdm.kubepodinventory** mdm.kubenodeinventory**>
+      type filter_inventory2mdm
+      custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westeurope,southafricanorth,centralus,northcentralus,eastus2,koreacentral,eastasia,centralindia,uksouth,canadacentral,francecentral,japaneast,australiaeast
+      log_level info
+     </filter>
+     #custom_metrics_mdm filter plugin for perf data from windows nodes
+     <filter mdm.cadvisorperf**>
+      type filter_cadvisor2mdm
+      custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westeurope,southafricanorth,centralus,northcentralus,eastus2,koreacentral,eastasia,centralindia,uksouth,canadacentral,francecentral,japaneast,australiaeast
```

Comment:
Dont have any background, but is this hardcoding needed?

Reply:
yes I believe it is needed as is. This hardcoding is part of OMS config: a/pkg/sync/v13/data/Deployment.apps/openshift-azure-logging/log-analytics-cluster-agent.yaml
And we already have it in our code base: https://github.com/openshift/openshift-azure/blob/master/pkg/sync/v14/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml#L67
it is longer in the new OMS agent version: https://github.com/microsoft/OMS-docker/compare/61729c6e3250910592d2e2b8391353f4c579d65a...6928f685f2c61f4ce3759a485c5ebfd4ddaf905d#diff-f4081688aa41659e0d49872cfbf54e1bR122